docs/docs/examples/vector_stores/DatabricksVectorSearchDemo.ipynb

```diff
@@ -64,6 +64,11 @@
     "    ServiceContext,\n",
     "    StorageContext,\n",
     ")\n",
+    "from llama_index.core.storage.docstore import SimpleDocumentStore\n",
+    "from llama_index.core.node_parser import HierarchicalNodeParser, get_leaf_nodes\n",
+    "from llama_index.core.retrievers import AutoMergingRetriever\n",
+    "from llama_index.core.query_engine import RetrieverQueryEngine\n",
```

Comment:
Can you move these imports to the `Advanced Usage` section? These are required only for the particular advanced use case.